modules/service/src/test/scala/lucuma/odb/graphql/OdbSuite.scala

```diff
@@ -375,7 +375,7 @@ abstract class OdbSuite(debug: Boolean = false) extends CatsEffectSuite with Tes
     query:     String,
     expected:  Either[List[String], Json],
     variables: Option[JsonObject] = None,
-    client:    ClientOption = ClientOption.Ws,
+    client:    ClientOption = ClientOption.Http,
```

Comment:
Are we going back here?  We might need to while `Ws` hides the problems.  In my PR I added a `ClientOption.Default` which is used as the default value in these methods to make it easy to switch.